app/index.html

```diff
@@ -85,12 +85,16 @@ <h2>Get Started in 1 Minute</h2>
       <div class="one-half column">
         <ol>
           <li>
-            <h4>Add your API on Kong</h4>
-            <p>After <a href="/install">installing</a> and starting Kong, use the Admin API on port 8001 to add a new API. Kong will route every incoming request with the specified public DNS to the associated target URL.<br/><br/></p>
+            <h4>Register a Service in Kong</h4>
+            <p>After <a href="/install">installing</a> and starting Kong, use the Admin API on port 8001 to add a new Service. Services represent upstream servers containing an API.</p>
           </li>
           <li>
-            <h4>Add Plugins on the API</h4>
-            <p>Then add extra functionality by using <a href="/plugins">Kong Plugins</a>. You can also create your own plugins.</p>
+            <h4>Add a Route to the Service</h4>
+            <p>Once you have a Service, add one (or many!) Route to it. Routes control how requests are transmitted to Services, as well as some basic transformations.</p>
```

Comment:
> ` one (or many!) Route `

I like the touch there, but "many Route" reads bad (the programmer in me gets irked at `one (or many!) Routes` but I think that's the correct form). Might be worth double-checking.